src/components/dialogs/generator-creation-dialog.js

```diff
@@ -183,6 +247,50 @@ const GeneratorCreationDialog = ({
         defaultValue: formValues?.ratedNominalPower,
     });
 
+    const [reactiveCapabilityCurve, reactiveCapabilityCurveField] =
+        useBooleanValue({
+            label: 'ReactiveCapabilityCurve',
+            validation: { isFieldRequired: true },
+            inputForm: inputForm,
+            defaultValue: formValues?.reactiveCapabilityCurve || true,
```

Comment:
I think you meant  `defaultValue: formValues?.reactiveCapabilityCurve ?? true` otherwise it simplifies to plain `true`.
This one is blocking for me.

Reply:
Done.